discovery-infra/install_cluster.py

```diff
@@ -208,8 +208,12 @@ def run_installation_flow_kube_api(
         kubeconfig_path,
 ):
     log.info("Approving agents")
-    for agent in cluster_deployment.wait_for_agents(nodes_number):
+    agents = cluster_deployment.wait_for_agents(nodes_number)
+    for agent in agents:
         agent.approve()
+    
+    log.info("Waiting for agent status verification")
+    agent.wait_for_agents_to_install(agents, nodes_number)
```

Comment:
Should call the static function
```suggestion
    Agent.wait_for_agents_to_install(agents, nodes_number)
```